compiler/parser/src/function.rs

```diff
@@ -76,8 +79,9 @@ pub fn parse_args(func_args: Vec<FunctionArgument>) -> Result<ArgumentList, Lexi
                 ));
             }
             None => {
-                // Allow starred args after keyword arguments.
-                if !keywords.is_empty() && !is_starred(&value) {
+                // Allow starred arguments after keyword arguments but
+                // not after double-starred arguments.
```

Comment:
Good catch, this does require a new error type though, CPython distinguishes between these:

```python
>>> foo(**{}, *[])
  File "<stdin>", line 1
    foo(**{}, *[])
              ^
SyntaxError: iterable argument unpacking follows keyword argument unpacking
```

Reply:
In CPython, this is tested in `test_kwargs_last3()` in `test_syntax.py`.